test/data/optional-migration-config.yml

```diff
@@ -0,0 +1,3 @@
+migration:
+  target:
+    product: bar
```

Comment:
This does not match with #81

Reply:
Yes, that's what I commented [here](https://github.com/SUSE/suse-migration-services/pull/88#issuecomment-494753645), once #81 is merged.

Reply:
Fixed.